hooks/command

```diff
@@ -1,16 +1,53 @@
 #!/bin/bash
 set -euo pipefail
 
+set +u
+# Check if the plugin is disabled
+if [[ ! -z "$BUILDKITE_PLUGIN_GIT_DIFF_CONDITIONAL_DISABLE_PLUGIN" ]]; then
+  echo "Plugin disable flag detected, passing entire pipeline to buildkite"
+  buildkite-agent pipeline upload $BUILDKITE_PLUGIN_GIT_DIFF_CONDITIONAL_DYNAMIC_PIPELINE
```

Comment:
SC2086: Double quote to prevent globbing and word splitting.
Try: buildkite-agent pipeline upload "$BUILDKITE_PLUGIN_GIT_DIFF_CONDITIONAL_DYNAMIC_PIPELINE"